src/connectors/r-a-d.io.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+Connector.artistTrackSelector = '#np';
+
+Connector.playButtonSelector = '#stream-play';
+
+Connector.playerSelector = '.dynamic-row > .col-md-6';
```

Comment:
There should also be mapping for the total duration:
```
Connector.durationSelector = '#progress-length';
```
and current time: 
```
Connector.currentTimeSelector = '#progress-current';
```

Reply:
Thanks. Ah, I see. I've comitted the necessary changes.